config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
So that the thing works on ipfs.io, localhost:8080, examplegateway.net, and from file:///